.github/workflows/stale.yml

```
@@ -0,0 +1,24 @@
+name: Mark stale issues and pull requests
```

Have you missed PR's or do we just want to do it issues first to test?

---

I was just going to to Issues first since we have a relatively good handle on PRs.

---

In which case (I know I know, I'm a stickler....)

```suggestion
name: Mark stale issues
```